PSKoans/Koans/Introduction/AboutNumbers.Koans.ps1

```diff
@@ -61,6 +61,15 @@ Describe 'Basic Number Types' {
 Describe "Banker's Rounding" {
 
     It 'rounds to nearest even number on .5' {
+        <#
+            Rounding
+
+            The rounding used in powershell is called "Rounding to Even" or "Banker's Rounding".
```

Comment:
```suggestion
            The rounding used in PowerShell is called "Rounding to Even" or "Banker's Rounding".
```

Reply:
Changed in 15d69579941977f82371cfe890290cf8fcd0168c